components/abtesting/optimizelyXExperiment/src/optimizely-x.js

```diff
@@ -0,0 +1,65 @@
+import ExperimentUseCase from './use-case'
+
+class OptimizelyXExperimentsService {
+  /**
+   * Get OptimizelyX SDK instance
+   * @return Promise<Object>
+   */
+  static async getSDK () {
+    return window && window.optimizely
+  }
+
+  /**
+   * Get info of an experiment
+   * @param {String} experimentId
+   * @return Promise<Object>
+   * @example
+   * {
+   *  id: '100',
+   *  isActive: true,
+   *  isInExperimentHoldback: true,
+   *  variation: {
+   *      id: '101',
+   *      name: 'Variation #1'
+   *  }
+   * }
+   */
+  static async getInfo (experimentId) {
+    let sdk = await this.getSDK()
```

Comment:
Well it's a mutable object I prefer not to. I know the asignation is constant. Not relevant anyways.